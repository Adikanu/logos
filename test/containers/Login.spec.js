import React from 'react';
import { mount } from 'enzyme';

import { Login } from 'containers/Login';

const mockDispatch = jest.fn();

function setup() {
  const props = {
    dispatch: mockDispatch,
    user: {
      authenticated: false,
      isAdmin: false,
    },
  };

  return mount(<Login {...props} />);
}

describe('Login', () => {
  const wrapper = setup(true);

  it('should be a Component', () => {
    expect(wrapper.instance() instanceof React.Component).toBe(true);
  });

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
