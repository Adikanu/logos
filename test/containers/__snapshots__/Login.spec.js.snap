exports[`Login should render properly 1`] = `"<div class=\"app__login app__route\"><div class=\"app__container\"><div class=\"app__login__wrapper\"><a href=\"#login\" class=\"btn btn-lg btn-primary btn-icon\" data-provider=\"github\"><i class=\"i-github\"></i><span>Login</span></a><a href=\"#login\" class=\"btn btn-lg btn-primary btn-icon\" data-provider=\"facebook\"><i class=\"i-facebook\"></i><span>Login</span></a></div></div></div>"`;
